### ISSUE TITLE
ImportError: No module named ruamel - `c7n_mailer` Run Error

### ISSUE THREAD

**Issue by mechastorm** [capitalone/cloud-custodian#1817]
I seem to be encountering this error when installing the latest version of c7n_mailer tool from master

This was my install and run logic
```
dir=$(pwd -P)
mailer_install_dir="cloud-custodian-mailer"
mailer_install_path="$dir/$mailer_install_dir"

git clone --branch master https://github.com/capitalone/cloud-custodian "$mailer_install_dir"
cd "$mailer_install_dir/"
virtualenv .venv
source .venv/bin/activate
cd tools/c7n_mailer
pip install --upgrade -r requirements.txt
python setup.py develop

export AWS_REGION=us-west-2
export AWS_PROFILE=elm
c7n-mailer --config "$dir/mailer/run.yml" --update-lambda
```

But the output I get from the command was
```
Traceback (most recent call last):
  File "/myworkspace/cloud-custodian-mailer/.venv/bin/c7n-mailer", line 9, in <module>
    load_entry_point('c7n-mailer==0.2', 'console_scripts', 'c7n-mailer')()
  File "/myworkspace/cloud-custodian-mailer/tools/c7n_mailer/c7n_mailer/cli.py", line 133, in main
    deploy.provision(mailer_config, functools.partial(session_factory, mailer_config))
  File "/myworkspace/cloud-custodian-mailer/tools/c7n_mailer/c7n_mailer/deploy.py", line 80, in provision
    archive = get_archive(config)
  File "/myworkspace/cloud-custodian-mailer/tools/c7n_mailer/c7n_mailer/deploy.py", line 44, in get_archive
    'redis')
  File "/myworkspace/cloud-custodian-mailer/.venv/local/lib/python2.7/site-packages/c7n/mu.py", line 72, in __init__
    self.add_modules(*modules)
  File "/myworkspace/cloud-custodian-mailer/.venv/local/lib/python2.7/site-packages/c7n/mu.py", line 91, in add_modules
    path = imp.find_module(module)[1]
ImportError: No module named ruamel
```

It seems to be package ruamel but I am not sure what part of the installation failed. 

This runs on 
Ubuntu 14.04 
Python 2.7.6

It this related to @whit537 changes from https://github.com/capitalone/cloud-custodian/pull/1417 ?

**Comment by JohnTheodore** [comment]
@mechastorm yes, I think it's related. see my comment on 1417

**Comment by whit537** [comment]
Try running `python setup.py develop` just before you `cd tools/c7n_mailer`. That should install the `master` version of `c7n`, which includes the namespace package support necessary for the mailer to find `ruamel` (i.e., #1677 #1710). As it is, you are almost certainly pulling the latest released version of `c7n` (0.8.27.0) under `pip install --upgrade -r requirements.txt`, and that does not have namespace package support.

Basically we need to cut a new release of `c7n` to support the updated mailer. cc: @kapilt 

P.S. Keep running `python setup.py develop` _after_ you `cd tools/c7n_mailer` and `pip install --upgrade -r requirements.txt` as well—you want to install both `c7n` and `c7n_mailer` from `master`.

**Comment by kapilt** [comment]
released 0.8.27.1
